common/lib/capa/capa/responsetypes.py

```diff
@@ -781,11 +877,21 @@ def setup_response(self):
         cxml = xml.xpath('//*[@id=$id]//choice', id=xml.get('id'))
 
         # contextualize correct attribute and then select ones for which
-        # correct = "true"
+        # correct = "true", with separate list for those with partial credit point values.
         self.correct_choices = [
             contextualize_text(choice.get('name'), self.context)
             for choice in cxml
-            if contextualize_text(choice.get('correct'), self.context) == "true"
+            if contextualize_text(choice.get('correct'), self.context) == 'true'
+        ]
+        self.partial_choices = [
```

Comment:
are you calculating this all the time even if partial credit wasn't selected? I guess I'm a bit confused here